code/controllers/subsystem/icon_smooth.dm

```diff
@@ -28,5 +28,15 @@ SUBSYSTEM_DEF(icon_smooth)
 			continue
 		smooth_icon(A)
 		CHECK_TICK
-
+	// Is this even needed, I dont know, lets do it anyway
+	for(var/turf/T in world)
+		if(T.smooth)
+			queue_smooth(T)
+		for(var/A in T)
+			var/atom/AA = A
+			if(AA.smooth)
+				queue_smooth(AA)
+	// Smooth those asteroids
+	for(var/turf/simulated/mineral/M in GLOB.mineral_turfs)
+		M.add_edges()
```

Comment:
@Fox-McCloud spline reticulation is just moved here

Reply:
I don't see any message.

Reply:
`SSicons` will show when its been done with the `Initialized Icon Smoothing subsystem in x seconds`, so it still kind of tells the user

Reply:
Missing the point.

Reply:
Are you saying the message is that important? The code is still there

Reply:
Yes, the message is that important.